docs/user-guide/environment-variables.md

```diff
@@ -62,6 +62,7 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 | USER_SHELL_BIN | sh | The user shell bin to run the build in. Can also be the full path such as `/bin/bash`.
 | GIT_SHALLOW_CLONE | true | **Options:** (`true`/`false`) <br><br>Shallow clones source repository.
 | GIT_SHALLOW_CLONE_DEPTH | 50 | Shallow clone with a history truncated to the specified number of commits
+| GIT_SHALLOW_CLONE_SINCE |  | Shallow clone with a history truncated starting from the specified datetime (inclusive). If set, this has priority over `GIT_SHALLOW_CLONE_DEPTH`.<br><br>This can be an absolute date such as `2019-04-01` or a relative date such as `4 weeks ago`.
```

Comment:
can we make it clear that it is using the option ` --shallow-since`? That way ppl can easily find docs about that option